shared/actions/chat2/index.js

```diff
@@ -858,14 +866,6 @@ const loadMoreMessagesSuccess = (results: ?Array<any>) => {
   return Saga.put(Chat2Gen.createSetConversationOffline({conversationIDKey, offline: res.offline}))
 }
 
-// If we're previewing a conversation we tell the service so it injects it into the inbox with a flag to tell us its a preview
-const previewConversation = (action: Chat2Gen.SelectConversationPayload) =>
```

Comment:
when I preview a channel in a team via the manage channels dialog, the `New conversation` row flickers in for ~0.5s before the row is added to the inbox and I'm dropped into preview mode.